variables.tf

```diff
@@ -88,6 +88,11 @@ variable "maintenance_window" {
   }
 }
 
+variable "migrated_users" {
```

Comment:
Not too fond of the `migrated` part actually. "Migration" is a transitional thing, not really a permanent thing. Perhaps we could do something like `admin_users`? Not ideal either, but best I think of right now. Other ideas are also welcome.

Reply:
`admin_users` could work, we just make sure that `postgres` is always in the list of users. Repositories using the module would need to update their code because we would likely change the output of the module into a mapped output `admin_user_password[name]`, extra work. This is probably the cleanest solution.

we could also name it `superusers` and avoid changing the output format of the module

Reply:
settled on `legacy_users`